compiler-rt/lib/scudo/standalone/allocator_common.h

```diff
@@ -53,7 +53,7 @@ template <class SizeClassAllocator> struct TransferBatch {
   void moveNToArray(CompactPtrT *Array, u16 N) {
     DCHECK_LE(N, Count);
     memcpy(Array, Batch + Count - N, sizeof(Batch[0]) * N);
-    Count -= N;
+    Count -= static_cast<u16>(Count - N);
```

Comment:
These two lines are not equivalent. Should that be "Count = ..."?

Reply:
oops, sorry my bad. I fixed that but didn't include it in the commit.

Reply:
Fixed. Thanks for catching it